Jellyfin.Server/Migrations/Routines/CreateUserLoggingConfigFile.cs

```diff
@@ -0,0 +1,93 @@
+            // 65fe243afbcc4b596cf8726708c1965cd34b5f68
+            @"{""Serilog"":{""MinimumLevel"":""Information"",""WriteTo"":[{""Name"":""Console"",""Args"":{""outputTemplate"":""[{Timestamp:HH:mm:ss}] [{Level:u3}] {ThreadId} {SourceContext}: {Message:lj} {NewLine}{Exception}""}},{""Name"":""Async"",""Args"":{""configure"":[{""Name"":""File"",""Args"":{""path"":""%JELLYFIN_LOG_DIR%//log_.log"",""rollingInterval"":""Day"",""retainedFileCountLimit"":3,""rollOnFileSizeLimit"":true,""fileSizeLimitBytes"":100000000,""outputTemplate"":""[{Timestamp:yyyy-MM-dd HH:mm:ss.fff zzz}] [{Level:u3}] {ThreadId} {SourceContext}:{Message} {NewLine}{Exception}""}}]}}],""Enrich"":[""FromLogContext"",""WithThreadId""]}}",
+            // 96c9af590494aa8137d5a061aaf1e68feee60b67
+            @"{""Serilog"":{""MinimumLevel"":""Information"",""WriteTo"":[{""Name"":""Console"",""Args"":{""outputTemplate"":""[{Timestamp:HH:mm:ss}] [{Level:u3}] [{ThreadId}] {SourceContext}: {Message:lj}{NewLine}{Exception}""}},{""Name"":""Async"",""Args"":{""configure"":[{""Name"":""File"",""Args"":{""path"":""%JELLYFIN_LOG_DIR%//log_.log"",""rollingInterval"":""Day"",""retainedFileCountLimit"":3,""rollOnFileSizeLimit"":true,""fileSizeLimitBytes"":100000000,""outputTemplate"":""[{Timestamp:yyyy-MM-dd HH:mm:ss.fff zzz}] [{Level:u3}] [{ThreadId}] {SourceContext}:{Message}{NewLine}{Exception}""}}]}}],""Enrich"":[""FromLogContext"",""WithThreadId""]}}",
+        };
+
+        /// <inheritdoc/>
+        public string Name => "CreateLoggingConfigHeirarchy";
+
+        /// <inheritdoc/>
+        public void Perform(CoreAppHost host, ILogger logger)
+        {
+            var logDirectory = host.Resolve<IApplicationPaths>().ConfigurationDirectoryPath;
+            var oldConfigPath = Path.Combine(logDirectory, "logging.json");
+            var userConfigPath = Path.Combine(logDirectory, Program.LoggingConfigFileUser);
+
+            // Check if there are existing settings in the old "logging.json" file that should be migrated
+            bool shouldMigrateOldFile = ShouldKeepOldConfig(oldConfigPath);
+
+            // Create the user settings file "logging.user.json"
+            if (shouldMigrateOldFile)
+            {
+                // Use the existing logging.json file
+                File.Copy(oldConfigPath, userConfigPath);
+            }
+            else
+            {
+                // Write an empty JSON file
+                File.WriteAllText(userConfigPath, EmptyLoggingConfig);
```

Comment:
Wasn't 100% sure about this. Maybe it would be better in this case if there was is no `logging.user.json` file created at all? New installs won't have this migration run so they also won't have the `logging.user.json` file created

Reply:
I think that works long-term, as long as we document that someone can add a `logging.user.json` to override settings.

Reply:
Indeed, a quick section in the documentation can cover this.